### ISSUE TITLE
Lightning集成Bug：每次运行显示事先已被调用导致复用

### ISSUE THREAD

**Issue by HelyaHsiung** [SwanHubX/SwanLab#1459]
### 确认项

- [x] 我已确认我遇到的问题在[文档](https://docs.swanlab.cn)中未有说明。
- [x] 我已在 [GitHub](https://github.com/SwanHubX/SwanLab/issues) 上搜索过类似或已有的问题。
- [x] 我已确认此问题并非由网络错误引起。

### 问题描述

在lightning与swanlab集成基础上，使用官方示例
```
from swanlab.integration.pytorch_lightning import SwanLabLogger

import importlib.util
import os

import pytorch_lightning as pl
from torch import nn, optim, utils
from torchvision.datasets import MNIST
from torchvision.transforms import ToTensor

# define any number of nn.Modules (or use your current ones)
encoder = nn.Sequential(nn.Linear(28 * 28, 128), nn.ReLU(), nn.Linear(128, 3))
decoder = nn.Sequential(nn.Linear(3, 128), nn.ReLU(), nn.Linear(128, 28 * 28))


# define the LightningModule
class LitAutoEncoder(pl.LightningModule):
    def __init__(self, encoder, decoder):
        super().__init__()
        self.encoder = encoder
        self.decoder = decoder

    def training_step(self, batch, batch_idx):
        # training_step defines the train loop.
        # it is independent of forward
        x, y = batch
        x = x.view(x.size(0), -1)
        z = self.encoder(x)
        x_hat = self.decoder(z)
        loss = nn.functional.mse_loss(x_hat, x)
        # Logging to TensorBoard (if installed) by default
        self.log("train_loss", loss)
        return loss

    def test_step(self, batch, batch_idx):
        # test_step defines the test loop.
        # it is independent of forward
        x, y = batch
        x = x.view(x.size(0), -1)
        z = self.encoder(x)
        x_hat = self.decoder(z)
        loss = nn.functional.mse_loss(x_hat, x)
        # Logging to TensorBoard (if installed) by default
        self.log("test_loss", loss)
        return loss

    def configure_optimizers(self):
        optimizer = optim.Adam(self.parameters(), lr=1e-3)
        return optimizer


# init the autoencoder
autoencoder = LitAutoEncoder(encoder, decoder)


# setup data
dataset = MNIST(os.getcwd(), train=True, download=True, transform=ToTensor())
train_dataset, val_dataset = utils.data.random_split(dataset, [55000, 5000])
test_dataset = MNIST(os.getcwd(), train=False, download=True, transform=ToTensor())

train_loader = utils.data.DataLoader(train_dataset)
val_loader = utils.data.DataLoader(val_dataset)
test_loader = utils.data.DataLoader(test_dataset)

swanlab_logger = SwanLabLogger(
    project="swanlab_example",
    experiment_name="example_experiment",
)

trainer = pl.Trainer(limit_train_batches=100, max_epochs=5, logger=swanlab_logger)


trainer.fit(model=autoencoder, train_dataloaders=train_loader, val_dataloaders=val_loader)
trainer.test(dataloaders=test_loader)
```

运行初始化swnalblogger时警告复用，接下来每一个step都会警告reuse，导致警告大量刷屏。
```
D:\miniconda3\envs\pytorch\lib\site-packages\swanlab\integration\pytorch_lightning.py:111: There is a swanlab experiment already in progress and newly created instances of `SwanLabLogger` will reuse this experiment. If this is not desired, call `swanlab.finish()` before instantiating `SwanLabLogger`.
```

### 必须的复现代码

核心代码是 …

复现步骤：
1. 打开复现
2. 点击 …
3. 看到错误： …

### 错误信息

(pytorch) PS D:\haiyang\Downloads\lightningEncoder> & D:/miniconda3/envs/pytorch/python.exe d:/haiyang/Downloads/lightningEncoder/main.py
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 9.91M/9.91M [00:02<00:00, 3.75MB/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 28.9k/28.9k [00:00<00:00, 86.7kB/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1.65M/1.65M [00:01<00:00, 1.02MB/s]
100%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 4.54k/4.54k [00:00<00:00, 1.13MB/s]
💡 Tip: For seamless cloud uploads and versioning, try installing [litmodels](https://pypi.org/project/litmodels/) to enable LitModelCheckpoint, which syncs automatically with the Lightning model registry.
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
HPU available: False, using: 0 HPUs
D:\miniconda3\envs\pytorch\lib\site-packages\pytorch_lightning\trainer\configuration_validator.py:68: You passed in a `val_dataloader` but have no `validation_step`. Skipping val loop.
swanlab: Tracking run with swanlab version 0.7.8
swanlab: Run data will be saved locally in D:\haiyang\Downloads\lightningEncoder\swanlog\run-20260228_143537-bg2flxxe2so1f1aojbg8j
swanlab: 👋 Hi helyahsiung,welcome to swanlab!
swanlab: Syncing run example_experiment to the cloud
swanlab: 🏠 View project at https://swanlab.cn/@helyahsiung/swanlab_example
swanlab: 🚀 View run at https://swanlab.cn/@helyahsiung/swanlab_example/runs/bg2flxxe2so1f1aojbg8j
D:\miniconda3\envs\pytorch\lib\site-packages\swanlab\integration\pytorch_lightning.py:111: There is a swanlab experiment already in progress and newly created instances of `SwanLabLogger` will reuse this experiment. If this is not desired, call `swanlab.finish()` before instantiating `SwanLabLogger`.
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name    | Type       | Params | Mode
-----------------------------------------------
0 | encoder | Sequential | 100 K  | train
1 | decoder | Sequential | 101 K  | train
-----------------------------------------------
202 K     Trainable params
0         Non-trainable params
202 K     Total params
0.810     Total estimated model params size (MB)
8         Modules in train mode
0         Modules in eval mode
D:\miniconda3\envs\pytorch\lib\site-packages\pytorch_lightning\trainer\connectors\data_connector.py:433: The 'train_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=19` in the `DataLoader` to improve performance.
Epoch 4: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100/100 [00:00<00:00, 197.95it/s, v_num=bg8j]`Trainer.fit` stopped: `max_epochs=5` reached.
Epoch 4: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████| 100/100 [00:00<00:00, 188.61it/s, v_num=bg8j]
D:\miniconda3\envs\pytorch\lib\site-packages\pytorch_lightning\trainer\connectors\checkpoint_connector.py:149: `.test(ckpt_path=None)` was called without a model. The best model of the previous `fit` call will be used. You can pass `.test(ckpt_path='best')` to use the best model or `.test(ckpt_path='last')` to use the last model. If you pass a value, this warning will be silenced.
Restoring states from the checkpoint path at .\swanlab_example\bg2flxxe2so1f1aojbg8j\checkpoints\epoch=4-step=500.ckpt
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]
Loaded model weights from the checkpoint at .\swanlab_example\bg2flxxe2so1f1aojbg8j\checkpoints\epoch=4-step=500.ckpt
D:\miniconda3\envs\pytorch\lib\site-packages\pytorch_lightning\trainer\connectors\data_connector.py:433: The 'test_dataloader' does not have many workers which may be a bottleneck. Consider increasing the value of the `num_workers` argument` to `num_workers=19` in the `DataLoader` to improve performance.
Testing DataLoader 0: 100%|███████████████████████████████████████████████████████████████████████████████████████████████████████| 10000/10000 [00:18<00:00, 539.97it/s] 
┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│         test_loss         │    0.05429396405816078    │
└───────────────────────────┴───────────────────────────┘
swanlab: Experiment example_experiment has completed
swanlab: 🏠 View project at https://swanlab.cn/@helyahsiung/swanlab_example
swanlab: 🚀 View run at https://swanlab.cn/@helyahsiung/swanlab_example/runs/bg2flxxe2so1f1aojbg8j
(pytorch) PS D:\haiyang\Downloads\lightningEncoder>

**Comment by HelyaHsiung** [comment]
I just remove the warning text from source code.
